about.html

```diff
@@ -0,0 +1,137 @@
+          </div>
+          <small class="small-desc">Please contact us per Email for any further questions about MUSON
+            2021</small>
+          <strong><u><a class="email-tag" href="#"><small
+                  class="desc-email">anivictor.chukwuemeka.avc@gmail.com</small></a></u></strong>
+        </div>
+      </div>
+    </section>
+
+    <section class="feature">
+      <article class="feature-info">
+        <div class="feature-logo">
+          <h3 class="">THE MUSICAL SOCIETY OF NIGERIA 25TH FESTIVAL 2021 Logo</h3>
+          <div class="line"></div>
+        </div>
+        <div class="logo-design">
+          <p class="">The logo Musical Society of Nigeria was decided through the logo
+            competition
+            from 8 June to 7
+            July.</p>
+        </div>
+        <div class="muson-logo logo-border"><img src="./images/home-logo.jpg" alt="muson logo"></div>
+      </article>
+
+      <article class="past-activities">
+        <div class="past-container">
+          <h3 class="past-heading">see the past MUSON conference</h3>
+          <div class="line"></div>
+          <p class="past-text">Take a look at the last two MUSON FESTIVAL which took place in Golden Royal,
+            Enugu and Transcorp Hilton, Abuja.</p>
+          <ul class="card-container">
+            <li class="card">
+              <a href="#">
+                <img class="card-images" src="./images/back-img.jpg" alt="Muson 2019">
+                <div class="card-block">
+                  <h3>2019</h3>
+                  <p class="text-center">MUSON Festival in Golden Royal, Enugu</p>
+                </div>
+              </a>
+            </li>
+
+            <li class="card">
+              <a href="#">
+                <img class="card-images" src="./images/thumbnail-1024x486.png" alt="Muson 2020">
+                <div class="card-block">
+                  <h3>2020</h3>
+                  <p class="text-center">MUSON Festival in transcorp hilton, Abuja</p>
+                </div>
+              </a>
+            </li>
+          </ul>
+        </div>
+      </article>
+    </section>
+
+    <section id="partners" class="partners">
+      <h2 class="partners-heading">Partners</h2>
+      <div class="line"></div>
+      <div class="partner-logo">
+        <img src="./images/image2vector.svg" alt="mozilla">
+        <img src="./images/google.svg" alt="google">
+        <img src="./images/naver.svg" alt="naver">
+        <img src="./images/tokopedia.png" alt="tokopedia">
+        <img src="./images/airbnb.svg" alt="airbnb">
+      </div>
+    </section>
+  </main>
+
+  <footer class="footer">
+    <div class="footer-container">
+      <a class="footer-logo-container" href="#"><img class="footer-logo" src="./images/home-logo.jpg"
+          alt="muson logo"></a>
+      <small class="footer-desc">&copy; Muson 2021. Some Rights</small>
```

Comment:
- [ ] Kindly note that the logo should be aligned to the left and text aligned to the right😎